ev3dev2/motor.py

```diff
@@ -1569,6 +1652,8 @@ def __init__(self, motor_specs, desc=None):
         for motor_port in sorted(motor_specs.keys()):
             motor_class = motor_specs[motor_port]
             self.motors[motor_port] = motor_class(motor_port)
+            self.motors[motor_port].ramp_up_sp = 0
+            self.motors[motor_port].ramp_down_sp = 0
```

Comment:
When we create a new Motor object should we be initializing most variables?  It hit me that if the previous robot that ran set ramp_up_sp or ramp_down_sp that could cause problems with MoveDifferential trying to drive in a perfect circle, etc.

Reply:
Why not use `Motor.reset()` instead?

Reply:
Will do that